src/main/kotlin/app/nush/bot/Config.kt

```diff
@@ -11,7 +11,11 @@ data class Config(
     val guildId: String,
     val memberRole: String,
     val excoRole: String,
-    val dev: Boolean = true
+    val dev: Boolean = true,
+    var userIsAppVentureMember: Boolean = false,
```

Comment:
This is not necessary

Reply:
That line refers to the [`excoChannelId` line](https://github.com/appventure-nush/appventure-bot/commit/b8afdad94f0070a45fa43bad6f3babe73de9987e#diff-aa9585738278c6cce14b50963e3b3f46R67)

Reply:
oh